LetterGradebyScoreAndProject/src/LetterGrade.java

```diff
@@ -0,0 +1,55 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+/*
+ * compute the score and then covert it to a letter grade
+ * the overall score = exam_score * 0.7 + project_score *.3 
+ */
+public class LetterGrade {
+	int examScore, projectScore;
+
+	public static void main(String[] args) {
+
+		LetterGrade lg = new LetterGrade();
+		try {
+			System.out.print("Enetr the Exam score = ");
+			InputStreamReader isr1 = new InputStreamReader(System.in);
+			BufferedReader br1 = new BufferedReader(isr1);
+			lg.examScore = Integer.parseInt(br1.readLine());
+			
+			System.out.print("Enetr the Project score = ");		
+			InputStreamReader isr = new InputStreamReader(System.in);
+			BufferedReader br = new BufferedReader(isr);
+			lg.projectScore = Integer.parseInt(br.readLine());	
+			
+			char grade = lg.letterGrade(lg.examScore, lg.projectScore);
+//	        System.out.println("The grade of is " + grade);	        
+		} catch (NumberFormatException ex) {
+			System.out.println("Not an integer !");
+		} catch (IOException e) {
+			e.printStackTrace();
+		}     
+
+	}  
+
+	public char letterGrade(int exam, int project) {

```

Comment:
should provide comment for this method

Reply:
OK

Reply:
@cxl1521 
The method comment has been added.